docs/modules/ROOT/pages/references/crds.adoc

```diff
@@ -40,6 +40,24 @@ This is a https://github.com/elastic/crd-ref-docs[generated] API documentation.
 |===
 
 
+[id="{anchor_prefix}-github-com-vshn-component-appcat-apis-v1-localobjectreference"]
+=== LocalObjectReference 
+
+LocalObjectReference contains enough information to let you locate the referenced object inside the same namespace.
```

Comment:
I would remove `enough`

Reply:
This is auto generated from the CRD and was copied directly from upstream k8s comment on `LocalObjectRefernce`. Is it OK for you if I leave that?

Reply:
no problem